### ISSUE TITLE
[FEATURE] Mobile Accessibility for hard of seeing

### ISSUE THREAD

**Issue by msprijatelj** [massif-press/compcon#2191]
**Is your feature request related to a problem? Please describe.**
CompCon is missing accessibility for screen readers like Apple VoiceOver, Supernova, or other screen readers on Mobile devices.

**Describe the solution you'd like**
I would like the mobile site to be screen reader friendly, including the ability to read tooltips, props, and/or alt text for icons that otherwise lack supporting text.

Having a screen-reader friendly version of the Compendium and/or its pages would also be helpful. Making sure that weapon, gear, and template descriptions don't "exist" in screen reader's perspective until selected, and then automatically read when they are.

**Describe alternatives you've considered**
Having a toggle to show/hide artwork may help screen readers have an easier time on the site, if alt text/props don't help enough.

**Additional context**
This info was provided by a member of Pilot NET who has a friend who is hard of seeing, which is preventing them from making the best use of CompCon.

**Comment by msprijatelj** [comment]
Additional post from Jan on PilotNet with suggestions/recommendations, lightly edited for clarity:

> By now I have two issues.
> 1. Some buttons are unlabelled. Primarily those used to navigating to a previous or the next pages. At least I think those buttons I'm using have that effect but I can't be sure because they don't have anything written. Labeling them shouldn't be very much trouble.
> 2. Things that are shown after hovering a mouse over them are problematic because it's impossible with a screen reader to view them. For example I encountered this with actions granted with talents. To view their details you should hover above them but it's impossible without a mouse. I believe that adding the option to view them after clicking on them should work.
